adaptors/backbone/base_collection.js

```diff
@@ -24,6 +26,69 @@ var BaseCollection = Backbone.Collection.extend({
     this.postInitialize();
   },
 
+  bindExposedEvent: function(event, childComponent) {
+    this.listenTo(childComponent.model, event, function() {
+      var args = Array.prototype.slice.call(arguments);
+      this.trigger.apply(this, [event].concat(args));
+      if (event.substr(0, 7) === 'change:') {
+        this.trigger('change', this);
+      }
+    });
+  },
+
+  /**
+   * Method for checking whether an object should be considered a model for
+   * the purposes of adding to the collection.
+   *
+   * Overriding to allow ComponentWidgets to be treated as models
+   *
+   * @param  {Object}  model Model to check
+   * @return {Boolean}
+   */
+  _isModel: function(model) {
+    return Backbone.Collection.prototype._isModel(model) || model instanceof ComponentWidget;
+  },
+
+  /**
+   * Binds events when a model is added to a collection
+   *
+   * @param {Object} model
+   * @param {Object} options
+   */
+  _addReference: function(model, options) {
+    Backbone.Collection.prototype._addReference.call(this, model, options);
+    if (ComponentWidget.isComponent(model) && model.exposedEvents) {
```

Comment:
ComponentWidget.isComponent checks that the object has a model property and that the model is a tungstenModel, so that wouldn't be necessary